source/scripting_v3/GTA/Entities/Vehicles/Vehicle.cs

```diff
@@ -1050,6 +1083,21 @@ public float ThrottlePower
 		/// </summary>
 		public float MaxTraction => Function.Call<float>(Hash.GET_VEHICLE_MAX_TRACTION, Handle);
 
+		/// <summary>
+		/// Reduces this <see cref="Vehicle"/>'s current grip level.
+		/// In game versions prior to v1.0.1604.0, levels 1-3 are identical.
+		/// </summary>
+		/// <param name="level">The level from 0-3 to reduce grip by, with 0 being no reduction and 3 being maximum reduction.</param>
+		public void ReduceGripLevel(int level)
+		{
+			Function.Call(Hash.SET_VEHICLE_REDUCE_GRIP, Handle, level != 0);
+
+			if (Game.Version >= GameVersion.v1_0_1604_0_Steam)
+			{
+				Function.Call(Hash.SET_VEHICLE_REDUCE_GRIP_LEVEL, Handle, level);
+			}
```

Comment:
Sorry, but have you tested that calling `SET_VEHICLE_REDUCE_GRIP` with the bool true has some effect without `SET_VEHICLE_REDUCE_GRIP_LEVEL`? I haven't confirmed, but I think this needs testing.

Reply:
I have indeed confirmed it. Here is a video: https://youtu.be/pS_9SZQ9DEQ